src/components/chat/speech-bubble/styles.module.scss

```
@@ -1,26 +1,56 @@
 .box {
   display: flex;
-  align-items: center;
-  margin: 1rem;
-  width: 28rem;
-  border: 0.1rem solid #000000;
-
-  img {
-    width: 3rem;
-    height: 3rem;
-    border-radius: 1.5rem;
-  }
+  align-items: flex-end;
+  gap: 1rem;
+  padding: 0 1rem;
 
-  .content {
-    .msg {
-      width: 20rem;
-      white-space: pre-wrap;
-      word-break: break-all;
-    }
+  & + & {
+    margin-top: 2rem;
   }
+}
 
-  .right-box {
-    display: flex;
-    align-items: center;
-  }
+.box.my-msg {
+  justify-content: flex-end;
+}
+
+.profile-image {
+  width: 5rem;
+  height: 5rem;
+  box-shadow: inset 0 0 0 3px #f1f1f1;
+  padding: 1rem;
+  border-radius: 50%;
+}
```

```suggestion
.profile-image {
  width: 5rem;
  height: 5rem;
  box-shadow: inset 0 0 0 3px #f1f1f1;
  padding: 1rem;
  border-radius: 50%;
   align-self: center;
}
```
   align-self: center;를 추가하면 프로필 사진만 가운데로 와서 더 자연스러울것 같습니다 !

---

```suggestion
.image-wrapper {
  overflow: hidden;
  width: 5rem;
  height: 5rem;
  border-radius: 50%;
  box-shadow: inset 0 0 0 3px #ccc;
  align-self: center;
}

.profile-image {
  width: 5rem;
  height: 5rem;
  padding: 1rem;
  transform: scale(1.7);
}r-radius: 50%;
}
```
아니면, img태그를 감싸는 div를 추가해서, 크기를 쬐끔 더 키워도 괜찮지 않을까유..?

---

넵 시도해보겠습니다.